crates/consensus/src/block/header.rs

```diff
@@ -214,10 +214,39 @@ impl Header {
     /// Returns `None` if `excess_blob_gas` is None.
     ///
     /// See also [Self::next_block_excess_blob_gas]
+    #[deprecated(
+        since = "0.8.1",
```

Comment:
not sure if  you are okay with this version the current one is 0.8.0 so for the next one is 0.8.1;

lmk if you want me to change or remove this